app-games/src/main/res/values-ru-rRU/strings.xml

```diff
@@ -264,22 +264,22 @@
   <!-- GenAi -->
   <string name="genai_bottom_navigation_genai_button">GenAI</string>
   <string name="genai_bottom_navigation_gamegenie_button">GameGenie</string>
-  <string name="genai_assistant_title">Assistant</string>
-  <string name="genai_me_title">Me</string>
-  <string name="genai_introduction_body">Hi! I\'m your AI assistant on everything related to apps and games discovery. Try with something like:</string>
-  <string name="genai_input_message_field">Type a message&#8230;</string>
-  <string name="genai_powered_by_ai_title">Powered by AI</string>
-  <string name="genai_example_1_body">I\'m looking for strategy games.</string>
-  <string name="genai_example_2_body">Please recommend me the best multiplayer games.</string>
-  <string name="genai_example_3_body">Do you know offline casual games?</string>
+  <string name="genai_assistant_title">Ассистент</string>
+  <string name="genai_me_title">Я</string>
+  <string name="genai_introduction_body">Привет! Я ваш помощник AI по всем вопросам, связанным с поиском приложений и игр. Попробуйте что-то вроде:</string>
+  <string name="genai_input_message_field">Введите сообщение&#8230;</string>
+  <string name="genai_powered_by_ai_title">На основе AI</string>
+  <string name="genai_example_1_body">Я ищу стратегические игры.</string>
+  <string name="genai_example_2_body">Порекомендуй мне, пожалуйста, лучшие многопользовательские игры.</string>
+  <string name="genai_example_3_body">Знаешь ли ты оффлайн казуальные игры?</string>
   <!-- Promotional Cards -->
-  <string name="promotional_now_with_appcoins">Now With AppCoins Payment System</string>
-  <string name="promotional_new_version">New Version</string>
-  <string name="promotional_on_going_event">Ongoing event</string>
-  <string name="promotional_new_in_aptoide">Now available</string>
-  <string name="promotional_soon_in_aptoide">Available sson</string>
-  <string name="promotional_notify_button">Notify me</string>
-  <string name="promotional_new_notification_title">The app you were waiting for!</string>
-  <string name="promotional_new_notification_body">%s is now available.</string>
-  <string name="promotional_new_active">You\'ll be notified when available.</string>
+  <string name="promotional_now_with_appcoins">Теперь с системой платежей AppCoins</string>
+  <string name="promotional_new_version">Новая версия</string>
+  <string name="promotional_on_going_event">Текущее событие</string>
+  <string name="promotional_new_in_aptoide">Недоступно</string>
```

Comment:
This one means "unavailable". I believe this is a mistake comparing to "Now available" string 🤔